application/cms/models.py

```diff
@@ -810,6 +816,8 @@ class Classification(db.Model):
         "Ethnicity", secondary=parent_association_table, back_populates="classifications_as_parent"
     )
 
+    __table_args__ = (UniqueConstraint(id, name="uq_classification_code"),)
```

Comment:
Updated to `uq`, now the new consistent format...

Reply:
https://github.com/racedisparityaudit/ethnicity-facts-and-figures-publisher/blame/master/migrations/versions/2018_10_04_code_is_id_for_classification.py#L34

😉 

This line added it as an undocumented constraint, and it's now depended on by quite a few other things. All I'm doing here is adding it to the model so that sqlalchemy knows about it. Seems like less faff than doing whatever juju we need to remove it.

Reply:
It looks like that line should have added it, but ~~it doesn't actually seem to exist in the database~~?

Oh, weird... it exists in prod database but not my local database, even though I just ran `pull_prod_data` which I thought should now copy all constraints.

Locally I have `data_source_frequency_of_release_id_fkey` which isn't in prod. But other than that the rest of the constraints seem identical.

Reply:
Weird, it's definitely in my local database
![screen shot 2018-12-07 at 21 52 35](https://user-images.githubusercontent.com/2920760/49674603-6ef7a180-fa6a-11e8-843f-dfa8e3c920d1.png)

`data_source_frequency_of_release_id_fkey` is in your local DB, but not prod, because this set of migrations adds it properly.

Reply:
Of couse this adds it :man_facepalming: 

But the missing unique constraint :man_shrugging: 
Have you run the `pull_prod_data` since it dumped and restored all the tables?